### ISSUE TITLE
WCAG-pagina 2.3.1 Drie flitsen of beneden drempelwaarde

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/documentatie#653]
Pagina schrijven voor WCAG Succescriterium 2.3.1 Drie flitsen of beneden drempelwaarde
Gerelateerd issue: https://github.com/nl-design-system/documentatie/issues/561

Niet vergeten:
De drempelwaarde bepalen van een gebied met 3 flitsen is nogal een berekening.
Zie: [Understanding SC 2.3.1 Three Flashes or Below Threshold (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/three-flashes-or-below-threshold.html)

Bij het kopje testen op de pagina [nldesignsystem.nl/wcag/2.3.1](https://nldesignsystem.nl/wcag/2.3.1) moet hierover informatie komen.
Ook onderzoeken hoe de Nederlandse onderzoeksbureau's hiermee omgaan.

Zie ook discussie op Slack/Fonteers/accessibility
https://fronteersnl.slack.com/archives/C6MGD72ER/p1727087071705839
